recipes/asm-blox

```diff
@@ -0,0 +1,3 @@
+(asm-blox :fetcher github
+          :repo "zkry/asm-blox"
+          :files ("*.el" (:exclude "asm-blox-tests.el")))
```

Comment:
I think this is a good fit -- it will automatically exclude the `-tests.el` file (and include the .info file, which isn't a bad thing since it's a packaging norm)
```suggestion
(asm-blox :fetcher github :repo "zkry/asm-blox")
```